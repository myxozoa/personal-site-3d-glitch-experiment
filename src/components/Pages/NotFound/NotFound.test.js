import React from 'react';
import { shallow } from 'enzyme';

import Component from './index';

describe('<NotFound />', () => {
  it('renders correctly', () => {
    const component = shallow(<Component />);

    expect(component).toMatchSnapshot();
  });
});