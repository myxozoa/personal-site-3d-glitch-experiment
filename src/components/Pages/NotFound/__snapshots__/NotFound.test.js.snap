// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> renders correctly 1`] = `
<div
  className="container"
>
  <h2
    className="glitch"
    data-text="Not_Found"
  >
    not_found
  </h2>
</div>
`;
